benchmark/build.gradle.kts

```diff
@@ -0,0 +1,36 @@
+plugins {
```

Comment:
Hmm no easy way to unify all that into sbt? is the Kotlin part an obstacle here?...

Reply:
There is https://github.com/pfn/kotlin-plugin although doesn't seem to be too _enthusiastically_ maintained ☹️